doc/list.rst

```diff
@@ -1723,6 +1723,18 @@ List of Available Rules
 
    Unused ``use`` statements must be removed.
 
+   Configuration options:
+
+   - | ``statements``
+     | List of statements which must be removed.
+     | Allowed types: ``array``
+     | Default value: ``['code','phpdoc_types','phpdoc_descriptions','annotations','comments']``
+
+   - | ``case_sensitive``
+     | Whether to case sensitive?
+     | Allowed types: ``bool``
+     | Default value: ``false``
```

Comment:
Symfony and CS fixer rulesets (at least, if this rule by default) should remove the imports if they was not found in the code with the same case.

Reply:
Thank you for your review. I did it according to the status quo.